code/game/atoms_movable.dm

```diff
@@ -1,5 +1,9 @@
+
+var/global/list/atoms_forced_gravity_processing = list()

```

Comment:
Why? Are global vars bad? I put it here because if this is ever reset without using the reset proc every atom that was in it will be left with their forced processing left on but not actually processing.